setup.cfg

```diff
@@ -11,7 +11,7 @@ show-response = 1
 minversion = 2.2
 norecursedirs = build docs/_build
 doctest_plus = enabled
-addopts = --fits
+addopts = --arraydiff-default-format=fits
```

Comment:
@bsipocz - ah, you need to also make sure ``--arraydiff`` is included here on its own before the ``--arraydiff-default-format`` - sorry for not spotting that sooner